.github/workflows/release.yml

```diff
@@ -48,5 +48,4 @@ jobs:
           #  https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets
           BLACKSMITH_FORGE_USERNAME: '${{ secrets.username }}'
           BLACKSMITH_FORGE_API_KEY: '${{ secrets.api_key }}'
-          PUPPET_GEM_VERSION: "~> 7.0"
```

Comment:
We should have always set this on a higher level, around line 25 so it also affects the `ruby/setup-ruby` actions. I consider that a bug in the previous implementation.

Reply:
It feels kind of like a single source of truth violation to set a value in the release workflow which might conflict with the modules default.   I agree that if the puppet gem version is to be explicitly declared in this workflow that it needs to be done prior to the bundle being installed.

The other position one could take is that modulesync_config is tightly coupled to this repo and changes to the Gemfile template require a new major version branch in this repo.

Reply:
Yes, but then it's duplicating info. It was as a workaround when Puppet 8 had issues

Reply:
My preference is [obviously] that we don't declare the gem version in this workflow.

Note that I am unable to merge in this repo.